src/Container/ApplicationFactory.php

```
@@ -144,6 +145,24 @@ public function __invoke(ContainerInterface $container)
     }
 
     /**
+     * Sets provided PHP settings, if any.
+     *
+     * @param ContainerInterface $container
+     */
+    private function setPhpSettings(ContainerInterface $container)
+    {
+        $config = $container->has('config') ? $container->get('config') : [];
+
+        if (!isset($config['php_settings']) || !is_array($config['php_settings'])) {
+            return;
+        }
+
+        foreach ($config['php_settings'] as $name => $value) {
+            ini_set($name, $value);
```

Should raise an exception if `ini_set` fails?